src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,90 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0b0 (9/15/2020)
+-------------------
+
```

Comment:
I'd add a brief description like "This restores support for Protobuf and for transitive excludes (using `!!` in the `dependencies` field)."

It'd be good to include that in the release email, too.